### PR TITLE
Add a role for OVN database adoption

### PR DESCRIPTION
**Pull request by booxter**

### CODE REVIEW

File: tests/vars.sample.yaml [hunk 1]
```diff
@@ -10,5 +10,8 @@ oc_header: |
 # External MariaDB IP for DB exports.
 external_mariadb_ip: CUSTOMIZE_THIS
 
+# External OVN DB IP for DB exports.
+external_ovndb_ip: CUSTOMIZE_THIS
```
Comment (jistr) [inline tests/vars.sample.yaml:14]: The PR looks good, we'll need to wire this into CI similarly as `external_mariadb_ip` to have a chance of passing CI.

https://review.rdoproject.org/cgit/rdo-jobs/tree/playbooks/data_plane_adoption/run_repo_tests.yaml#n12

File: tests/roles/ovn_adoption/tasks/.main.yaml.swp
Comment (booxter) [inline tests/roles/ovn_adoption/tasks/.main.yaml.swp:1]: Ouch, this should be removed (and probably .gitignore updated).

**Comment by booxter** [comment]
CI will not pass until https://review.rdoproject.org/r/c/rdo-jobs/+/48929 is merged.

**Comment by booxter** [comment]
recheck

**Review by jistr — APPROVED** [review]
Looks good, thanks!

**Comment by booxter** [comment]
recheck

**Pull request merged**